README.md

```diff
@@ -63,6 +59,12 @@ control "aws-1" do
 end
 ```
 
+### Running your profile
+
+Then use `inspec exec my-profile` to execute your new profile.
+
+Our future intent is to support an `aws` target for InSpec/Train, so you may also pass credentials `inspec exec my-profile -t aws://accesskey:secret@region`.
```

Comment:
If you like, link to https://github.com/chef/train/issues/229

Reply:
done